Makefile

```diff
@@ -116,8 +116,8 @@ lint-fix:
 
 .PHONY: po
 po:
-	$(VIRTUAL_ENV)/bin/python manage.py makemessages --all --extension html,email,py --ignore venv --ignore node_modules
-	$(VIRTUAL_ENV)/bin/python manage.py makemessages --all -d djangojs --ignore venv --ignore node_modules
+	$(VIRTUAL_ENV)/bin/python manage.py makemessages --all --extension html,email,py --no-obsolete --ignore venv --ignore node_modules
```

Comment:
we don't want the obsolete strings at the end of the file, right? this way we don't have to delete them manually

Reply:
No, but we would like to notice when we loose strings... I always check there for the js strings.

Reply:
hmm, but you would see that in the diff anyway? I guess it's a bit harder to spot though. Now that we found out the reaason for the disappearing js translations it shouldn't happen anymore hopefully? In any case: should I remove this?

Reply:
Yes, I would like to keep the obsolete strings to make sure we don't loose translations accidentally. But maybe we only need to be sure for the js files. So only remove it from the next line?
What do you think @Rineee and @phillimorland ?

Reply:
I hadn't thought of the obsolete strings, I mostly check in the diff but maybe leaving them for js until we have brought the codebase up to scratch would be good

Reply:
ok, I removed the commit